tests/ui/specs/VirtualList/VirtualGridList/onScroll/VirtualGridList-GT-30066-specs.js

```diff
@@ -0,0 +1,36 @@
+const Page = require('../VirtualGridListPage');
+
+describe('Navigate with Channel Up/Down key with translate mode', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should navigate via Page Down/Up key on scrollbar [GT-30066]', function () {
```

Comment:
I think this TC will not be completely automated. Because animation should be checked.
If there is a label related to "partial automation", how about use it instead of "Enact_ui_tests_automated"?

Reply:
We have used a label 'Enact_ui_tests_part_automated'. I will use this label.